R/rcpp.R

```diff
@@ -51,3 +87,25 @@ use_src <- function() {
 
   invisible()
 }
+
+create_makevars <- function(path, package, settings = NULL) {
```

Comment:
Should this be called `use_makevars()` and match the signature of the other `use_*` functions? I don't think we should export it at this point, but we should consider the API.